tests/exposures/test_csv_trans.py

```diff
@@ -20,6 +24,7 @@
 
 class CsvTrans(unittest.TestCase):
     @given(integers(min_value=1, max_value=10))
```

Comment:
This is just a suggestion but I think that any arguments in test cases whose data are generated by `hypothesis.given` but should be named, e.g.
    ...
    @given(chunk_size=integers(min_value=1, max_value=10))
    def test_source_to_canonical(self, chunk_size):
    ...

It is a bit easier to read the test case that way as you know which arguments are using which data generation strategy.

Also I would put the `settings` decorator at the top, above `given`.